Sources/Valet/Valet.swift

```diff
@@ -603,6 +629,53 @@ extension Valet {
         return containsObject
     }
 
+    /// Migrates objects matching the input query into the receiving Valet instance.
+    /// - Parameters:
+    ///   - query: The query with which to retrieve existing keychain data via a call to SecItemCopyMatching.
+    ///   - compactMap: A closure that transforms a key:value pair from the raw pair currently in the keychain into a key:value pair we'll insert into the destination Valet. Returning `nil` from this closure will cause that key:value pair not to be migrated.
```

Comment:
Should this mention the `preventMigration` functionality?

Reply:
```suggestion
    ///   - compactMap: A closure that transforms a key:value pair from the raw pair currently in the keychain into a key:value pair we'll insert into the destination Valet. Returning `nil` from this closure will cause that key:value pair not to be migrated. Returning `VALMigratableKeyValuePairOutput.preventMigration` will prevent migrating any key:value pairs.
```